flink-ml-lib/src/main/java/org/apache/flink/ml/common/util/quantile/QuantileSummary.java

```diff
@@ -36,6 +37,7 @@
  * algorithm proposed in the paper: "Space-efficient Online Computation of Quantile Summaries" by
  * Greenwald, Michael and Khanna, Sanjeev. (https://doi.org/10.1145/375663.375670)
  */
+@TypeInfo(QuantileSummaryTypeInfoFactory.class)
```

Comment:
Would it be simpler to put both `QuantileSummary` and `QuantileSummaryTypeInfoFactory` directly under the package 
`common/util`?

Note that `QuantileSummaryTypeInfoFactory` should be an internal class that users should not need to access directly.